src/gui/MainFrame.cpp

```diff
@@ -915,7 +915,7 @@ void MyMainFrame::SetSingleParticleWorkflow(bool triggered_by_gui_event) {
             }
         }
         current_workflow = cistem::workflow::single_particle;
-        current_project.RecordCurrentWorkflowInDB(current_workflow);
+        if (current_project.is_open == true) current_project.RecordCurrentWorkflowInDB(current_workflow);
```

Comment:
@twagner9 

It looks like your editing environment is not setup properly to use the clang format specification found at [main project directory]/.clang-format

If so, this line would have been automatically formatted as:
```c++
if (current_project.is_open == true) 
    current_project.RecordCurrentWorkflowInDB(current_workflow);
```

That formatting detail aside, this is a correct fix. Here is a shot of my current file

![image](https://user-images.githubusercontent.com/3077528/196517513-8a479784-0201-41da-9fca-ea8140fa42d5.png)

(A less important stylistic detail - comparing bools to true or false isn't necessary, but is allowed, unlike single line conditionals.)

Reply:
* Don't worry about the formatting in this PR, it will update when I (or @jojoelfe or anyone else) pulls it in. Please do sort out the clang-format issue for the future though. If you have questions, drop me an email.
*